### DISCUSSION TITLE
In ground, swamfeeder will be detached when the latched target is stunned or presses the ragdoll key.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#13616] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you press ragdoll key when your character is in ground, all swamfeeders will instantly be detached from your body.
This is not a bug, but I think it may not be intentional because it's too simple to do (you just need to press ragdoll key).
Maybe it would be better to keep latching whether the target is stunned or not.

### Reproduction steps

1. Spawn some swamfeeders
2. Wait for swamfeeders latch you
3. Press ragdoll key, swamfeeders will be detached

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by zhu-rengong** [comment]
Not sure if it's a bug, but the issue has been moved to discussion, which means this is allowed: in water, pressing the ragdoll key, the swamfeeder will not be detached, but it can be on the ground.
I'm making a monster mod and I can't seem to find any way to prevent the player from breaking away from the latcher by pressing the ragdoll key.

**Comment by Regalis11** [comment]
This does seem like a bug, we moved it to discussions because [we are no longer collecting bug reports through the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13590).

**Comment by ubertpendragon** [comment]
[Seasnek.zip](https://github.com/FakeFishGames/Barotrauma/files/14716428/Seasnek.zip)

Another latching monster if you want to test (seasnekbiter)

**Comment by Jasontti** [comment]
Tested this again and it does happen, more easily demonstrated when feeders are latched to extremidies. On body they seems to stay on better.

**Comment by Regalis11** [comment]
Thank you for the report! This has now been addressed in our private repo, fix coming up in the next unstable build and the upcoming Winter Update.
